packages/@sfcc-bff/basketapi/src/api/schema/basketResolvers.ts

```diff
@@ -5,30 +5,37 @@
     For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
 */
 import apollo from 'apollo-server';
-import Cart from '../models/Cart';
+import Basket from '../models/Basket';
 import { getCommerceClientConfig } from '@sfcc-core/apiconfig';
-import { getUserFromContext } from '@sfcc-core/core-graphql';
+import { getUserFromContext, AppContext } from '@sfcc-core/core-graphql';
 import CommerceSdk from 'commerce-sdk';
-import { core } from '@sfcc-core/core';
+import { core, Config } from '@sfcc-core/core';
+
 const { ApolloError } = apollo;
 const logger = core.logger;
-const getBasketClient = async (config, context) => {
+
+const getBasketClient = async (config: Config, context: AppContext) => {
     const clientConfig = getCommerceClientConfig(config);
-    clientConfig.headers.authorization = (
-        await getUserFromContext(context)
-    ).token;
+    clientConfig.headers.authorization =
+        (await getUserFromContext(context))?.token ?? '';
```

Comment:
`??` is the null coalescing operator: https://medium.com/better-programming/how-to-use-the-null-coalescing-operator-in-javascript-15adefe7c655